spec/plans/environment.fmf

```diff
@@ -0,0 +1,17 @@
+summary: Environment variables
+
+description:
+    Specifies environment variables available in all steps. Plugins
+    need to include these environment variables while running
+    commands or other programs.
+
+example: |
+    /plan:
+        environment:
+            - name: KOJI_TASK_ID
```

Comment:
What about using dictionary for the key-value pairs?

    environment:
        KOJI_TASK_ID: 42890031
        RELEASE: f33

Would be more concise and a bit more clear.

Reply:
yeah, I did this for a reason (limitation on key values), but might not make much sense :)

Reply:
changing

Reply:
fixed